jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -52,7 +52,10 @@ OPENJ9_ENABLE_DDR       := @OPENJ9_ENABLE_DDR@
 COMPILER_VERSION_STRING := @COMPILER_VERSION_STRING@
 USERNAME                := @USERNAME@
 OPENJDK_SHA             := @OPENJDK_SHA@
-OPENJDK_TAG             := @OPENJDK_TAG@
+
+# include openjdk-tag.gmk file
+openjdk_tag_include_file := $(TOPDIR)/closed/openjdk-tag.gmk
+include $(openjdk_tag_include_file)
```

Comment:
Unless we anticipate reusing `openjdk_tag_include_file` elsewhere, I don't think it deserves to be captured in a macro; that also makes the comment superfluous.

Reply:
Also, I think this should use `SRC_ROOT` instead of `TOPDIR` for consistency with other makefiles.

Reply:
SRC_ROOT / TOPDIR is a good point, will check...
checked...! jdk8 is a bit mixed, but SRC_ROOT is predominent!
jdk11+ is TOPDIR